packages/contracts-bedrock/scripts/deploy/DeployConfig.s.sol

```diff
@@ -51,6 +51,7 @@ contract DeployConfig is Script {
     uint256 public l2GenesisBlockGasLimit;
     uint32 public basefeeScalar;
     uint32 public blobbasefeeScalar;
+    uint32 public daFootprintGasScalar;
```

Comment:
Despite being present in the diff [here](https://github.com/ethereum-optimism/optimism/pull/12166/files#diff-e01bdaa0f060aea6713d30cafb64636481979288096116353e594fe2c17e8848), it appears that the `operatorFee*` parameters never made it through to [`develop`](https://github.com/ethereum-optimism/optimism/blame/develop/packages/contracts-bedrock/scripts/deploy/DeployConfig.s.sol).